fractions/javaee/messaging/src/main/java/org/wildfly/swarm/messaging/MessagingFraction.java

```diff
@@ -104,7 +104,10 @@ public MessagingFraction server(String childKey, EnhancedServerConsumer config)
     private EnhancedServer findOrCreateDefaultServer() {
         EnhancedServer server = (EnhancedServer) subresources().server("default");
         if (server == null) {
-            server("default", EnhancedServer::enableInVm);
+            server("default", defaultServer -> {
+                defaultServer.enableInVm();
+                defaultServer.enableRemote(); // TODO ???
```

Comment:
I don't want this enabled by default, but I want an easy way to enable it. I didn't feel like creating a `messaging-remote` fraction :-) What would be the best course of action here?

Reply:
Why not a customizer that checks for presence of `ejb-remote`?

Reply:
Sounds reasonable, thanks!